frontend/src/app/settings/search-history-settings/search-history-settings.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'ia-search-history-settings',
+    templateUrl: './search-history-settings.component.html',
+    styleUrls: ['./search-history-settings.component.scss']
+})
+export class SearchHistorySettingsComponent implements OnInit {
+
+    constructor() { }
+
+    ngOnInit(): void {
+    }
```

Comment:
Tiniest hill to die on.
```suggestion
export class SearchHistorySettingsComponent {

    constructor() { }
```